packages/amplication-client/src/Resource/ResourceHome.scss

```diff
@@ -28,6 +28,19 @@ $circle-badge-border: 2px;
     align-items: center;
     justify-content: center;
   }
+  &__header:hover::before {
+    z-index: 1;
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: #ffffff99;
```

Comment:
@lokeswaran-aj  Please change the hard-coded color value to var,
Example: var(--black20);

Reply:
I could not find an existing variable, so I have create a variable in the top and used it.